pkg/kubectl/cmd/rollout/rollout_pause.go

```diff
@@ -75,7 +73,7 @@ func NewCmdRolloutPause(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	return cmd
 }
 
-func (o *PauseConfig) CompletePause(f *cmdutil.Factory, cmd *cobra.Command, out io.Writer, args []string) error {
+func (o *PauseConfig) RunPause(f *cmdutil.Factory, cmd *cobra.Command, out io.Writer, args []string) error {
```

Comment:
`run` methods shouldn't require commands or factories.  Rather than hold up the rest of the work on this, splitting it out is fine.